### DISCUSSION TITLE
Inputting Custom Electrostatic Potential into Parakeet

### DISCUSSION THREAD

**Discussion by david-yw-t** [rosalindfranklininstitute/amplus-digital-twin#72] [Q&A]
Hi Parakeet team,

Is it possible to input our own electrostatic potential map (for instance, one generated from our own simulation process) into Parakeet and then proceed to obtain the exit wave and other related outputs, instead of using a molecular structure in the form of a PDB file?

Thanks!

**Comment by jmp1985** [comment]
Hi @david-yw-t 

Unfortunately, currently, this isn't easy to do but it is something that I really want to implement (see https://github.com/rosalindfranklininstitute/parakeet/issues/46) when I find time!

I guess you have a 3D array containing the atomic potential at each voxel. How big is your volume? The main issue is with holding everything in memory for large volumes which is why I did this on the fly in parakeet.

Best wishes

James
